spec.html

```diff
@@ -32989,13 +32987,11 @@
 
       <!-- es6num="24.1.2.1" -->
       <emu-clause id="sec-arraybuffer-length">
-        <h1>ArrayBuffer ( _length_ )</h1>
-        <p>ArrayBuffer called with argument _length_ performs the following steps:</p>
+        <h1>ArrayBuffer ( [ _length_ ] )</h1>
```

Comment:
No, that's not true. Unless you're also advocating to change the length of Object (length is 1, required 0), Function (length is 1, required 0), Array (length is 1, required 0), String (length is 1, required 0), Number (length is 1, required 0), Boolean (length is 1, required 0), Date (length is 7, required 0), etc. etc. :-/